dts/bindings/pinctrl/pincfg-node.yaml

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2020, Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+description: Generic pin configuration schema
+
+child-binding:
+    description: |
+      Many data items that are represented in a pin configuration node are
+      common and generic. Pin control bindings should use the properties
+      defined below where they are applicable; not all of these properties are
+      relevant or useful for all hardware or binding structures. Each
+      individual binding document should state which of these generic
+      properties, if any, are used, and the structure of the DT nodes that
+      contain these properties.
+
+      This is based on Linux, documentation:
+        https://www.kernel.org/doc/Documentation/devicetree/bindings/pinctrl/pincfg-node.yaml
+
+    properties:
+      bias-disable:
+        required: false
+        type: boolean
+        description: disable any pin bias
+
+      bias-high-impedance:
+        required: false
+        type: boolean
+        description: high impedance mode ("third-state", "floating")
+
+      bias-bus-hold:
+        required: false
+        type: boolean
+        description: latch weakly
+
+      bias-pull-up:
+        required: false
+        type: boolean
+        description: |
+          pull resistive up the pin. Can be used together with bias-pull-down.
+
+      bias-pull-down:
+        required: false
+        type: boolean
+        description: |
+          pull resistive down the pin. Can be used together with bias-pull-up.
+
+      bias-pull-pin-default:
+        required: false
+        type: boolean
+        description: use pin-default pull state.
+
+      drive-push-pull:
+        required: false
+        type: boolean
+        description: drive actively high and low
+
+      drive-open-drain:
+        required: false
+        type: boolean
+        description: drive with open drain (hardware AND).
+
+      drive-open-source:
+        required: false
+        type: boolean
+        description: drive with open source (hardware OR).
+
+      drive-strength:
+        required: false
+        type: int
+        description: sink or source at most X mA
+
+      drive-strength-microamp:
+        required: false
+        type: int
+        description: sink or source at most X uA
```

Comment:
```suggestion
        description: sink or source current in μA
```

Reply:
maybe `maximum sink or source current in μA` since it said `at most`.